### ISSUE TITLE
Access denied to local storage with Torus

### ISSUE THREAD

**Issue by allemanfredi** [Web3Modal/web3modal#93]
I'm trying to use Torus but i receive the following error when i try to connect with it.

```
DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
```

**Comment by crisgarner** [comment]
This is a problem with a new version of Torus, fix is on it's way

**Comment by chaitanyapotti** [comment]
This has been fixed in torus versions >=v0.2.11

**Comment by chaitanyapotti** [comment]
@pedrouid can you close this.?

**Comment by pedrouid** [comment]
Fixed on `@toruslabs/torus-embed@>=0.2.11`
